<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>App Diccionario</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutMain">
    <item>
     <layout class="QHBoxLayout" name="tabBarLayout">
      <item>
       <widget class="QToolButton" name="toolBtnDiccionario">
        <property name="text">
         <string>Diccionario</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolBtnSRS">
        <property name="text">
         <string>SRS</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageDictionary">
       <layout class="QVBoxLayout" name="layoutDictionary">
        <item>
         <widget class="QLineEdit" name="lineEditSearch">
          <property name="placeholderText">
           <string>Buscar...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="searchResultList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageDetails">
       <layout class="QVBoxLayout" name="layoutDetails">
        <item>
         <widget class="QToolButton" name="btnBack">
          <property name="text">
           <string>←</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="detailsScroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="detailsContainer">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>754</width>
             <height>434</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="detailsLayout">
            <item>
             <widget class="QLabel" name="labelMainWord">
              <property name="styleSheet">
               <string>
font-size: 32px;
font-weight: bold;
padding: 6px;
               </string>
              </property>
              <property name="text">
               <string>—</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelReadings">
              <property name="styleSheet">
               <string>
font-size: 20px;
color: #444;
padding-left: 6px;
               </string>
              </property>
              <property name="text">
               <string>Lecturas aquí…</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="lineSeparator1">
              <property name="frameShape">
               <enum>QFrame::Shape::HLine</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Sunken</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelSensesTitle">
              <property name="styleSheet">
               <string>
font-size: 18px;
font-weight: 600;
padding: 6px 0 0 6px;
               </string>
              </property>
              <property name="text">
               <string>Significados</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listSenses">
              <property name="styleSheet">
               <string>
font-size: 16px;
padding: 4px;
               </string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacerBottom">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSRS">
       <layout class="QVBoxLayout" name="layoutSRS">
        <item>
         <widget class="QLabel" name="labelSRS">
          <property name="text">
           <string>Contenido SRS</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
